### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ErdbeerSaft**
Penis needs to be added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,6 @@ pflichtenheft Spaceshooter Pasi,Kev,Michi,Alex
 ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
 
 ⭕️
+
+
+Penis
```
Comment (Cr4zyBr41n) [inline README.md]: NSFBIB

**Review by F0UNT41N — DISMISSED** [review]
I <3 Miguel he is da real bro

**Review by Cr4zyBr41n — DISMISSED** [review]
we need to chang eto P*nis to make it SFBIB

**Review by Cr4zyBr41n — COMMENTED** [review]
is k now

**Review by Cr4zyBr41n — APPROVED** [review]
is k now

**Review by F0UNT41N — CHANGES_REQUESTED** [review]
no.

**Pull request closed without merging**